### DISCUSSION TITLE
What are the default values for Event Location and Region?

### DISCUSSION THREAD

**Discussion by delapuente** [Qiskit/qiskit.org#2389] [General]
Airtable `Event Location` and `Region` (`place` and `location` in the UI) do not have default values. Although they may seem mandatory, there is nothing enforcing a value in the Airtable so these fields could come empty. In those cases. What are the default values?

Right now, the default value for an absent `Event Location` is the `Region` if that exists. If not, it is nothing. For the `Region`, there is no fallback: if it is not provided, it shows nothing.

**Comment by lerongil** [comment]
Suggestion to add TBD as a fall back location. One other suggestion is to makeup a fake location (nice to have) but preferred TBD.

**Comment by delapuente** [comment]
@y4izus now we are switching to the new table, is this solved or mitigated somehow?

**Comment by y4izus** [comment]
Just to clarify:
- As Salva said, the default value for an absent `Event Location` is the `Region`, but `Region` default in `TBD` when it hasn't value
- The Region is never shown on the card, we only show the Location
- The Region is used for the filter, so, an event with a Region that has `TBD` or a value that doesn't match with the checkbox options, won't be shown as a result of the filter research

If you agree with this behavior, we can close the issue

**Comment by delapuente** [comment]
I'm fine with this. If the Event team has a better suggestion we can always reopen this issue.
